ci/script.sh

```diff
@@ -21,6 +21,9 @@ case $TARGET in
 		sudo apt-get install -y cmake pkg-config libssl-dev
 
 		cargo test --all --release --locked "$@"
+
+		# Test for staking without `equalize` default feature
+		(cd srml/staking/; cargo test --tests tests --no-default-features --features std)
```

Comment:
I'm not sure this is the way to go, but currently no test are triggered without the equalize feature it seems

Reply:
cc @gabreal

note: now that equalize is on by default, if there isn't a clean way to enable both, we should stick to `default-features = ['equalize']` to have this version tested.

Reply:
I'm not sure whether I understand correctly. Why is there a new ci folder when the scripts that are in use by the gitlab ci are located at `scripts` folder? The ci should in general not install any packages but only run the tests in a pre-configured environment.

`cargo test` is run by the ci from [here](https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml#L126) and if there should be additional tests it may make sense to introduce an additional step (for speed reasons).

Reply:
oh I don't know about `ci/script.sh` file to be honest it was just there, I might wanted to add this additional run of cargo test `(cd srml/staking/; cargo test --tests tests --no-default-features --features std)`

Reply:
but this is not related to this PR just we don't test staking modules without the "equalize" feature

Reply:
@thiolliere would #2995 be working for you?

Reply:
yes :-)